presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -64,11 +65,11 @@ public void configure(Binder binder)
     @Provides
     @Singleton
     @InternalBaseJdbc
-    public static JdbcClient createJdbcClientWithStats(JdbcClient client)
+    public JdbcClient createJdbcClientWithStats(JdbcClient client)
```

Comment:
Why remove `static`? Looks like leftover, doesn't belong to `Use different loggers for JDBC client invocation logging` commit

Reply:
because `catalogName` is a field, not accessible with a static method.